### PR TITLE
Publish Code Engine/Paketo post

### PR DESCRIPTION
**Pull request by dmuelle**
#2675 

Looks good on [staging](https://blogs-staging-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/blog/2022/11/14/using-codeengine-with-paketo-buildpack-for-liberty.html)

Screen Shot 2022-11-14 at 1 49 50 PM

### CODE REVIEW

File: posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+layout: post
+title: "Deploy Java applications to IBM Cloud Code Engine using the Paketo Buildpack for Liberty"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars.githubusercontent.com/u/29545669?v=4
+author_github: https://github.com/kevin-ortega
+seo-title:  Deploy Java applications to IBM Cloud Code Engine (ICCE) using the Paketo Buildpack for Liberty - OpenLiberty.io
+seo-description: Learn to deploy your applications to ICCE easily with the Paketo Buildpack for Liberty
+blog_description: "Learn to deploy your applications to ICCE easily with the Paketo Buildpack for Liberty"
+open-graph-image: https://openliberty.io/img/blog/code_engine_build_step_details.png
```
Comment (kinueng) [inline posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc:11]: My suggestion next time is to try using a relative path for the `open-graph-image` value. As of right now, the image used here is not on `openliberty.io` yet.

I usually will test the SEO fields (like the `og:image`) in Slack to see the metadata that gets pulled into preview cards. Here is an example of me DMing myself the link to test the Slack's URL preview cards. Notice an image is missing.
image

Comment (kinueng) [reply posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc:11]: Woops, I did not give a suggestion on a relative path. Here is what I would of tried `open-graph-image: /img/blog/code_engine_build_step_details.png`.

Comment (mbroz2) [inline posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc:11]: I would have suggested using [code_engine_build_strategy.png](https://github.com/OpenLiberty/blogs/pull/2746/files#diff-f654c4f7cbedd759fea54bf522d90fa0d1504ad6c068c5055c83e914101ec2f0)  instead as it seems more interesting

**Review by kinueng — APPROVED** [review]
LGTM

I made a small comment that I do not think should block this from being merged. A suggestion to better improve the review process for our social metadata. 

In the case of this PR, I check the URL to confirm it should work in `prod`

File: posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc (hunk 1 above)
Comment (mbroz2) [reply posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc:11]: from my recollection, relative paths use to not work for this.... but maybe things have changed or I'm misremembering

File: posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+Kevin Ortega <https://github.com/kevin-ortega>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+You can use link:https://www.ibm.com/cloud/code-engine[IBM Cloud Code Engine] to easily deploy Java applications by using the link:https://openliberty.io/blog/2022/04/01/cloud-native-liberty-buildpack.html[Paketo Buildpack for Liberty] with your container images or source code.
+
+This post provides instructions on how to use the Paketo Buildpack for Liberty with Code Engine. To learn more about using Code Engine, see link:https://cloud.ibm.com/docs/codeengine?topic=codeengine-getting-started[Getting started with IBM Cloud Code Engine].
+
+With the Paketo Buildpack for Liberty, you can use Open Liberty or WebSphere Liberty to run your application.
```
Comment (mbroz2) [inline posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc:24]: Remove this sentence.
```suggestion
```

File: posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+Code Engine provides the next level of easy application deployment and mature management of critical business applications in IBM Cloud.  Move your applications from the Cloud Foundry liberty-for-java buildpack to a cloud native solution using Code Engine and the Paketo Buildpack for Liberty.
+
+== What do you need?
+* An https://www.ibm.com/cloud[IBM Cloud] Account
+* Your application source
+
+To deploy container images, you'll also need:
+
+* The link:https://buildpacks.io/docs/tools/pack/[pack CLI]
+* https://www.docker.com/[Docker]
```
Comment (mbroz2) [inline posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc:36]: @kevin-ortega, can alternatives like podman be used, or is docker a must?

Comment (mbroz2) [reply posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc:36]: If podman can be used, we should call that out.

File: posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+* An externally accessible container registry that can host your images.
+
+For details on deploying your application from repository source code from the Code Engine console, see  link:https://cloud.ibm.com/docs/codeengine?topic=codeengine-app-source-code#deploy-app-source-code[Deploying your app from repository source code from the console].
+
+For details on deploying your application from repository source code with the CLI, see  link:https://cloud.ibm.com/docs/codeengine?topic=codeengine-app-source-code#deploy-app-source-code-cli[Deploying your app from repository source code with the CLI].
```
Comment (mbroz2) [inline posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc:41]: The links of these sentences are redundant with the part before them (and also there's an extra space after 'see').  This should be restructured to remove the redundancy. perhaps something like 
For details on deploying your application from repository source code using:
* link:https://cloud.ibm.com/docs/codeengine?topic=codeengine-app-source-code#deploy-app-source-code[Code Engine console]
* link:https://cloud.ibm.com/docs/codeengine?topic=codeengine-app-source-code#deploy-app-source-code-cli[CLI]

File: posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+.To summarize, two important configuration steps are required to use the Paketo Buildpack for Liberty in Code Engine:
+. Create a `.buildenv` file that defines the  `BP_JAVA_APP_SERVER=liberty` environment variable.
+. Set the listening port to 9080
+
+== Other environment variables you can set in the `.buildenv` file
+BP_LIBERTY_INSTALL_TYPE::
+Specifies the  link:https://github.com/paketo-buildpacks/liberty#install-types[Install type] of Liberty. Possible values are `ol` (Open Liberty) or  `wlp` (WebSphere Liberty).  Open Liberty is the default.
+
+BP_LIBERTY_PROFILE::
+Specifies which liberty profile to install. Valid profiles for Open and WebSphere Liberty are documented link:https://github.com/paketo-buildpacks/liberty#profiles[in the buildpacks documentation].
```
Comment (mbroz2) [inline posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc:111]:
```suggestion
Specifies which liberty profile to install. Valid profiles for Liberty are documented link:https://github.com/paketo-buildpacks/liberty#profiles[in the buildpacks documentation].
```

File: posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+drwxr-xr-x  11 kevin  staff   352 Oct 20 08:53 target
+```
+
+.To summarize, two important configuration steps are required to use the Paketo Buildpack for Liberty in Code Engine:
+. Create a `.buildenv` file that defines the  `BP_JAVA_APP_SERVER=liberty` environment variable.
+. Set the listening port to 9080
+
+== Other environment variables you can set in the `.buildenv` file
+BP_LIBERTY_INSTALL_TYPE::
+Specifies the  link:https://github.com/paketo-buildpacks/liberty#install-types[Install type] of Liberty. Possible values are `ol` (Open Liberty) or  `wlp` (WebSphere Liberty).  Open Liberty is the default.
```
Comment (mbroz2) [inline posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc:108]:
```suggestion
Specifies the link:https://github.com/paketo-buildpacks/liberty#install-types[Install type] of Liberty.  Open Liberty (value of `ol`) is the default.
```

**Pull request merged**